### PR TITLE
Change unresolved BasePlotter intersphinx refs to Plotter

### PR DESCRIPTION
**Pull request by adeak**
I noticed that `BasePlotter` method references don't resolve in the docs:

screenshot from outside plotting.py showing red, unresolved pyvista.BasePlotter.subplot() reference

screenshot from within plotting.py showing red, unresolved BasePlotter.update reference

Switching the references to `Plotter` should fix the issue (existing links referring to `pyvista.Plotter`, or within `plotting.py` to `Plotter`, get resolved correctly).

### CODE REVIEW

File: examples/02-plot/multi-window.py [hunk 1]
```diff
@@ -116,8 +116,9 @@
 
 plotter = pv.Plotter(shape=shape, row_weights=row_weights, col_weights=col_weights, groups=groups)
 
+###############################################################################
 # A grouped subplot can be activated through any of its composing cells using
-# the :func:`pyvista.BasePlotter.subplot` function.
+# the :func:`pyvista.Plotter.subplot` method.
```
Comment (adeak) [inline examples/02-plot/multi-window.py:121]: This block looked like it was supposed to be outside the code blocks. I've added the break header, but we'll have to check the doc build artifact to make sure the result doesn't look weird in some way (I'm mostly concerned by having a `Plotter` created in the first block but shown in the second).

File: examples/02-plot/multi-window.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 ###############################################################################
 # This example shows how to create a multi-window plotter by specifying the
 # ``shape`` parameter.  The window generated is a two by two window by setting
-# ``shape=(2, 2)``. Use the :func:`pyvista.BasePlotter.subplot` function to
+# ``shape=(2, 2)``. Use the :func:`pyvista.Plotter.subplot` function to
```
Comment (tkoyama010) [inline examples/02-plot/multi-window.py:15]:
```suggestion
# ``shape=(2, 2)``. Use the :func:`pyvista.Plotter.subplot` method to
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: examples/02-plot/multi-window.py (hunk 2 above)
Comment (adeak) [reply examples/02-plot/multi-window.py:15]: Good catch, thanks!

File: examples/02-plot/multi-window.py (hunk 1 above)
Comment (adeak) [reply examples/02-plot/multi-window.py]: The original change broke the doc build so I've made this into a proper comment with no sphinx markup.

**Review by tkoyama010 — APPROVED** [review]
LGTM except for comments about block.

**Pull request merged**